general-exploration-landcover_io.ipynb

```diff
@@ -38,6 +38,9 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    study_bbox
</pre><p>I am not sure but it may be easier to create directly the study_box with the coordinates e.g.</p><pre class="ql-syntax" spellcheck="false">study_bbox_coordinates = [[x - xd, y - yd], [x - xd, y + yd], [x + xd, y + yd], [x + xd, y - yd]]
#view bounding box
study_bbox_coordinates
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Environmental-DS-Book/general-exploration-landcover_io/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Yes, I agree, this approach saves lines of code below. I have made edits accordingly</p>

Reply:
<p>closing conversation, resolved</p>